### PR TITLE
pkp/pkp-lib#562 Implement automatic review reminders

### PR DESCRIPTION
**Pull request by bozana**
https://github.com/pkp/pkp-lib/issues/562

### CODE REVIEW

File: dbscripts/xml/upgrade.xml [hunk 1]
```diff
@@ -71,6 +71,8 @@
 		<code function="htmlifyEmailTemplates" /><!-- Bug #8929 -->
 		<data file="dbscripts/xml/upgrade/1.2.0.0_update.xml" />
 		<code function="convertQueries" />
+		<code function="installEmailTemplate" key="REVIEW_REQUEST_REMIND_AUTO" locales="de_DE,el_GR,en_US,es_ES,fr_CA,pt_BR" />
+		<code function="installEmailTemplate" key="REVIEW_REQUEST_REMIND_AUTO_ONECLICK" locales="de_DE,el_GR,en_US,es_ES,fr_CA,pt_BR" />
 		<note file="docs/release-notes/README-BEACON" />
 	</upgrade>
```
Comment (asmecher) [inline dbscripts/xml/upgrade.xml]: I think this will also need the task added for upgrades, no?

Comment (bozana) [inline dbscripts/xml/upgrade.xml]: What do you mean? :-\

**Comment by bozana** [comment]
@asmecher, I considered the task in the update, rebased and actualized the submodule reference... THANKS A LOT!!!

File: locale/en_US/emailTemplates.xml [hunk 2]
```diff
@@ -339,7 +389,7 @@ Submission URL: <a href="{$submissionReviewUrl}">{$submissionReviewUrl}</a><br /
 Please confirm your ability to complete this vital contribution to the work of the press. I look forward to hearing from you.<br />
 <br />
 {$editorialContactSignature}]]></body>
-		<description>This email is automatically sent when a reviewer's due date elapses (see Review Options under Management &gt; Settings &gt; Workflow &gt; Review) and one-click reviewer access is enabled. Scheduled tasks must be enabled and configured (see the site configuration file).</description>
+		<description>This email is automatically sent when a reviewer's due date elapses (see Review Options underSettings &gt; Workflow &gt; Review) and one-click reviewer access is enabled. Scheduled tasks must be enabled and configured (see the site configuration file).</description>
```
Comment (asmecher) [inline locale/en_US/emailTemplates.xml]: Typo!

**Pull request merged**